test/vm/vm_provider/virtualbox/Vagrantfile

```diff
@@ -0,0 +1,33 @@
+Vagrant.configure("2") do |config|
+
+  config.vm.provider "virtualbox" do |v|
```

Comment:
Maybe it's good to ensure the `vagrant-vbguest` is installed @incertum? WDYT?

Reply:
Done and also added a general dependency check for all conditions I could think of atm ...

Reply:
Thank you @incertum!!